docs/open-api/lineage.yaml

```diff
@@ -26,9 +26,9 @@ info:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
you can remove the above codes since `openapi.yaml` already has these

Reply:
removed